model/src/main/kotlin/config/FileArchiverConfiguration.kt

```diff
@@ -33,7 +33,7 @@ data class FileArchiverConfiguration(
     /**
      * Configuration of the [FileStorage] used for archiving the files.
      */
-    val storage: FileStorageConfiguration
+    val storage: FileStorageConfiguration,
```

Comment:
Please create the left-over trailing comma. (Even if it's actually valid Kotlin syntax!)